swiftsimio/visualisation/slice.py

```diff
@@ -393,11 +427,20 @@ def slice_gas(
         Data field to be projected. Default is mass. If None then simply
         count number of particles 
     
-    parallel : bool
+    parallel : bool, optional
         used to determine if we will create the image in parallel. This 
         defaults to False, but can speed up the creation of large images 
         significantly at the cost of increased memory usage.
 
+    rotation_matrix: np.array, optional
```

Comment:
Does it provide a projection along the z axis? I don't think that's the case :).

I think we also need to add a comment to the `slice` parameter to make sure that it's clear that the z that will be used is in the rotated frame.